root/components/area-map.tt

```diff
@@ -0,0 +1,100 @@
+<div id="largemap"></div>
+<script type="text/javascript">//<![CDATA[
+
+    [%~ DEFAULT places=entities ~%]
+    var map,
```

Comment:
Wrap the code in an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression) like this to avoid creating global variables on the page:

```JavaScript
<script type="text/javascript">//<![CDATA[
(function () {
    // code goes here
}());
//]]></script>
```